### ISSUE TITLE
Stream does not restart after background on IOS Safari

### ISSUE THREAD

**Issue by cadalt** [videojs/video.js#6447]
## Description
When you are watching a m3u8 stream on ios safari and background the page for a few minutes, when you bring the page back to the foreground you get an error, or the play buttons are unresponsive.  You have to refresh or use the scrubber to play the video

## Steps to reproduce
1. Go to the following url, and view the video on your mobile browser, iOS Safari: 
https://www.wesh.com/article/63-years-after-their-first-dance-high-school-sweethearts-reunite-and-get-married/30923910
2. While the video is playing, lock your cell phone, and put it down for 5 minutes
3. Log back into your phone, and open iOS Safari

## Results
The player shows the following error: "The media playback was aborted due to a corruption problem or because the media used features your browser did not support." or the play control does not play the video. You have to use the scrubber to start the video playing

### Expected
The player resumes where it left off, or you can press play to resume the video

### Actual
The player shows the following error: "The media playback was aborted due to a corruption problem or because the media used features your browser did not support." or the player is an unresponsive black box. You have to refresh the page to get the video back

### Error output
Sometimes you get the error: The media playback was aborted due to a corruption problem or because the media used features your browser did not support.

### versions
we reproduced on various iphones from 7 up including iPhone XS 13.3 and iPhone 8 Plus 13.2.3
#### videojs
7.5.4
#### browsers
safari
#### OSes
ios
### plugins
videojs-contrib-ads, videojs-dock, videojs-ima, videojs-markers, videojs-overlay

**Comment by Patrickwong8015** [comment]
Please don't send for me again

welcome[bot] <notifications@github.com> 于 2020年2月15日周六 上午4:11写道：

**Comment by gkatsev** [comment]
Yeah, this is a deep safari issue that we haven't had the chance to create a work around for.
Webkit issues:
https://bugs.webkit.org/show_bug.cgi?id=203415 and https://bugs.webkit.org/show_bug.cgi?id=195452
